File: 250216/첫 번째와 두 번째 교환/exchange-1st-and-2nd.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String str = br.readLine();
        char strArray[] = str.toCharArray();
        char first = strArray[0];
        char second = strArray[1];
        for (int i = 0; i < strArray.length; i++) {
            if (strArray[i] == first)
                strArray[i] = second;
            else if (strArray[i] == second)
                strArray[i] = first;
        }
        System.out.println(String.valueOf(strArray));
    }
}

File: 250216/첫 번째와 두 번째 교환/exchange-1st-and-2nd.py
string = input()

strArray = list(string)

first = strArray[0]
second = strArray[1]

for i in range(len(string)):
    if (strArray[i] == first):
        strArray[i] = second
    elif (strArray[i] == second):
        strArray[i] = first

print("".join(strArray))